### PR TITLE
Add new transkribus model for swe-wikisource

### PR DESCRIPTION
**Pull request by kolakachi**
Bug:T352658

### CODE REVIEW

File: tests/Controller/OcrControllerTest.php [hunk 1]
```diff
@@ -27,6 +27,7 @@ class OcrControllerTest extends OcrTestCase {
 	 * @covers OcrController::getLangs
 	 * @param string[] $getParams
 	 * @param string[] $expectedLangs
+	 * @return void
```
Comment (samwilson) [inline tests/Controller/OcrControllerTest.php]: This is correct, but why include it in this PR?

File: composer.lock
Comment (samwilson) [inline composer.lock:1]: Is this Composer update needed in order to add the new model? It might be better to add it in a separate PR.

**Review by samwilson — COMMENTED** [review]
Looks fine, but I'm just wondering about mixing the different updates in one PR.

File: tests/Controller/OcrControllerTest.php (hunk 1 above)
Comment (kolakachi) [reply tests/Controller/OcrControllerTest.php]: Linting was failing for me.

File: composer.lock
Comment (kolakachi) [reply composer.lock:1]: This was missed. will take it out.

File: tests/Controller/OcrControllerTest.php (hunk 1 above)
Comment (samwilson) [reply tests/Controller/OcrControllerTest.php]: Ah right. Does that mean linting is failing on this in current main as well?

Comment (kolakachi) [reply tests/Controller/OcrControllerTest.php]: yes it's failing on main as well.

`  1x: Method "Psr\Log\LoggerAwareInterface::setLogger()" might add "void" as a native return type declaration in the future. Do the same in implementation "MediaWiki\OAuthClient\Client" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in OcrControllerTest::testGetLang with data set #0 from App\Tests\Controller`

Comment (samwilson) [reply tests/Controller/OcrControllerTest.php]: Oh interesting, is that showing as an error for you? It's a deprecation warning for me, perhaps you have some other env var set to elevate it?

The warnings I see are:

```
Remaining indirect deprecation notices (4)

  1x: Method "Psr\Log\LoggerAwareInterface::setLogger()" might add "void" as a native return type declaration in the future. Do the same in implementation "MediaWiki\OAuthClient\Client" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in OcrControllerTest::testGetLang with data set #0 from App\Tests\Controller

  1x: The "Symfony\Bridge\Doctrine\Logger\DbalLogger" class implements "Doctrine\DBAL\Logging\SQLLogger" that is deprecated Use {@see \Doctrine\DBAL\Logging\Middleware} or implement {@see \Doctrine\DBAL\Driver\Middleware} instead.
    1x in OcrControllerTest::testGetLang with data set #0 from App\Tests\Controller

  1x: Method "Symfony\Component\Console\Command\Command::execute()" might add "int" as a native return type declaration in the future. Do the same in child class "Doctrine\Bundle\DoctrineBundle\Command\CreateDatabaseDoctrineCommand" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in OcrControllerTest::testGetLang with data set #0 from App\Tests\Controller

  1x: Method "Symfony\Component\Console\Command\Command::execute()" might add "int" as a native return type declaration in the future. Do the same in child class "Doctrine\Bundle\DoctrineBundle\Command\DropDatabaseDoctrineCommand" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in OcrControllerTest::testGetLang with data set #0 from App\Tests\Controller
```

Which I agree we should fix :-) but they should probably be in separate PRs.

**Comment by samwilson** [comment]
It's weird that this is now failing. It looks like something's changed upstream, perhaps it's this issue? https://github.com/symfony/symfony/issues/52844
